### PR TITLE
TextRender: contents from Element instance and getTree convert options

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

**Comment by stdweird** [comment]
requires https://github.com/quattor/CCM/pull/51

File: pom.xml [hunk 1]
```diff
@@ -51,6 +51,9 @@
       <name>Ben Jones</name>
       <email>ben.jones@morganstanleylcom</email>
```
Comment (jrha) [inline pom.xml]: **.**com?

**Comment by stdweird** [comment]
i'm having serious second thoughts about this PR. it's mixing CCM and CAF, and this breaks the dependency chain.
i'm probably going to move all CCM code to new subclass CCM::TextRender (as i would like to expose even more CCM details to TT usage; it would blow up completely).

**Comment by stdweird** [comment]
There will be a CAF and a CCM PR to replace this

**Comment by stdweird** [comment]
CCM part now in https://github.com/quattor/CCM/pull/51

**Pull request closed without merging**